R/family-lists.R

```diff
@@ -611,3 +611,31 @@
     ybounds = c(-Inf, Inf), closed = c(NA, NA)
   )
 }
+
+.family_xbetax <- function() {
+    list(
+        links = c(
+            "logit", "probit", "probit_approx", "cloglog",
+            "cauchit", "softit", "identity", "log"
+        ),
+        dpars = c("mu", "phi", "kappa"),
+        type = "real",
+        ybounds = c(0, 1),
+        closed = c(TRUE, TRUE),
+        ad = c("weights", "subset", "index"),
+        include = "fun_xbetax.stan",
+        prior = function(dpar, link = "identity", ...) {
+        if (dpar == "kappa") {
+            if (link == "identity") {
+                return("gamma(0.01, 0.01)")
+            }
+            else {
```

Comment:
else should be on the same line as the if closing bracket }

Reply:
commit https://github.com/paul-buerkner/brms/pull/1698/commits/3f3f8e9d3ba9af3a0a2caea0f4151d0045aaa046